forch/forch_metrics.py

```diff
@@ -85,6 +107,31 @@ def get_metrics(self, path, params):
         """Return metric list in printable form"""
         return generate_latest(self._reg).decode('utf-8')
 
+    def _get_path_metrics(self, path):
```

Comment:
also, just realized you're perpetuating the wrong abstraction here.... this is just a web proxy, it has nothing to do with metrics... so likely should be pulled out into its own separate class.

Reply:
Makes sense. Created new file.